openmetadata-service/src/main/java/org/openmetadata/service/alerts/generic/GenericWebhookPublisher.java

```diff
@@ -90,7 +90,7 @@ public void sendAlert(EventResource.ChangeEventList list)
       Throwable cause = ex.getCause();
       if (cause != null && cause.getClass() == UnknownHostException.class) {
         LOG.warn("Invalid webhook {} endpoint {}", webhook.getName(), webhook.getEndpoint());
-        setErrorStatus(attemptTime, Response.Status.INTERNAL_SERVER_ERROR.getStatusCode(), "UnknownHostException");
+        setErrorStatus(attemptTime, null, "UnknownHostException");
```

Comment:
we should have an error status no? it should be 400

Reply:
will update it